### PR TITLE
fixes (#6801) Preview Recipe JSON doesn't scroll to recipe json

### PR DESCRIPTION
**Pull request by cherish2003**
### Because
Page not scrolling to the recipe json part when clicked

### This commit
- Removed the Entire webpage url and mentioned only the id

### CODE REVIEW

**Comment by cherish2003** [comment]
@yashikakhurana kindly review the code and let me know any changes are required

File: experimenter/experimenter/nimbus-ui/src/components/SidebarActions/index.tsx [hunk 1]
```diff
@@ -191,7 +190,7 @@ export const SidebarActions = ({
 
         {recipeJson && (
           <Link
-            to={`${BASE_PATH}/${slug}#recipe-json`}
+            to={`#recipe-json`}
```
Comment (yashikakhurana) [inline experimenter/experimenter/nimbus-ui/src/components/SidebarActions/index.tsx:193]: good catch 👏

**Review by yashikakhurana — APPROVED** [review]
@cherish2003 thank you for working on this, little fix but impactful :)

**Comment by cherish2003** [comment]
@yashikakhurana mam this issue is still not yet completed

**Pull request merged**